homeassistant/components/n26/sensor.py

```diff
@@ -0,0 +1,294 @@
+ATTR_ACC_OWNER_LAST_NAME = "owner_last_name"
+ATTR_ACC_OWNER_GENDER = "owner_gender"
+ATTR_ACC_OWNER_BIRTH_DATE = "owner_birth_date"
+ATTR_ACC_OWNER_EMAIL = "owner_email"
+ATTR_ACC_OWNER_PHONE_NUMBER = "owner_phone_number"
+
+ICON_ACCOUNT = 'mdi:currency-eur'
+ICON_CARD = 'mdi:credit-card'
+ICON_SPACE = 'mdi:crop-square'
+
+
+async def async_setup_platform(
+        hass, config, async_add_entities, discovery_info=None):
+    """Set up the N26 sensor platform."""
+    api = hass.data[COMPONENT_DOMAIN][API]
+    account_info = hass.data[COMPONENT_DOMAIN][ACCOUNT_INFO]
+    balance = api.get_balance()
+    limits = api.get_account_limits()
+    account_statuses = api.get_account_statuses()
+
+    sensor_entities = [N26Account(
+        api, account_info, account_statuses, limits, balance)]
+
+    cards = hass.data[COMPONENT_DOMAIN][CARDS]
+    for card in cards:
+        sensor_entities.append(N26Card(api, balance["iban"], card))
+
+    spaces = hass.data[COMPONENT_DOMAIN][SPACES]
+    for space in spaces["spaces"]:
+        sensor_entities.append(N26Space(api, space))
+
+    async_add_entities(sensor_entities)
+
+
+class N26Account(Entity):
+    """Sensor for a N26 balance account.
+
+    A balance account contains an amount of money (=balance). The amount may
+    also be negative.
+    """
+
+    def __init__(self, client, account_info,
+                 account_statuses, limits, balance) -> None:
+        """Initialize a N26 balance account."""
+        self._client = client
+        self._account_info = account_info
+        self._account_statuses = account_statuses
+        self._limits = limits
+        self._balance = balance
+
+    @property
+    def should_poll(self) -> bool:
+        """Return True.
+
+        Data needs to be polled from the bank servers.
+        """
+        return True
+
+    def update(self) -> None:
+        """Get the current balance and currency for the account."""
+        self._account_info = self._client.get_account_info()
+        self._account_statuses = self._client.get_account_statuses()
+        self._limits = self._client.get_account_limits()
+        self._balance = self._client.get_balance()
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the entity."""
+        return self._balance["iban"]
+
+    @property
+    def name(self) -> str:
+        """Friendly name of the sensor."""
+        return "n26_%s" % self._balance["iban"].lower()
+
+    @property
+    def state(self) -> float:
+        """Return the balance of the account as state."""
+        return self._balance["bankBalance"]
+
+    @property
+    def unit_of_measurement(self) -> str:
+        """Use the currency as unit of measurement."""
+        return self._balance["currency"]
+
+    @property
+    def device_state_attributes(self) -> dict:
+        """Additional attributes of the sensor."""
+        attributes = {
+            ATTR_IBAN: self._balance.get("iban"),
```

Comment:
Do we really need put those sensitive information to attributes? Such as birth date and account number 

Also remembered all attributes will be saved in the DB, is that necessary to monitor those never changed attributes?